internal/onchain/wallet/wallet.go

```diff
@@ -867,6 +886,55 @@ func (wallet *Wallet) GetTransactions(limit, offset uint64) ([]*onchain.WalletTr
 	return transactions, nil
 }
 
+func (wallet *Wallet) BumpTransactionFee(txId string, satPerVbyte float64) (string, error) {
+	if wallet.subaccount == nil {
+		return "", ErrSubAccountNotSet
+	}
+	outputs, err := wallet.getUnspentOutputs(*wallet.subaccount, false)
+	if err != nil {
+		return "", err
+	}
+
+	var limit uint64 = 30
+	var offset uint64
+	var found any
+	for {
```

Comment:
https://github.com/BoltzExchange/boltz-client/pull/388/files/5d6311620787302ec857d51d23bddd69fcd6564c#r2010371501
Did you not push a commit?